src/components/Questionnaire/QuestionTypes/MedicationRequestQuestion.tsx

```diff
@@ -182,6 +183,12 @@ export function MedicationRequestQuestion({
       questionnaireResponse.question_id,
     );
   };
+  const sortedMedications = [...medications].sort((a, b) => {
```

Comment:
<img width="384" alt="image" src="https://github.com/user-attachments/assets/39513ae2-112f-4a42-a5e0-7b779fb203e0" />

Current logic would result in "active", "ended" and then everything else.